apps/site/src/app/apply/sections/Form/AgeInformation.tsx

```diff
@@ -0,0 +1,46 @@
+import RequiredAsterisk from "../Components/RequiredAsterisk";
+import SimpleRadio from "../Components/SimpleRadio";
```

Comment:
Chore: relative imports are usually more cumbersome to manage. We can use the alias for the src directory and use absolute imports.

Reply:
I'm not entirely sure how to do this or what this is referring to to be honest, so I might not be able to implement this

Reply:
We can specify imports from the top-level down using `@` to reference our `src` directory instead of using `../` to traverse relative paths. Refer to [About.tsx](https://github.com/HackAtUCI/irvinehacks-site-2024/blob/main/apps/site/src/app/(home)/sections/About/About.tsx) for an example.
```tsx
import { thing } from "@/lib/components"
```

Reply:
should we also do this to files that are in the same directory (./)?

Reply:
No, for the files that are in the same directory, just use `./`.

Reply:
got it, then this should be set